app/assets/stylesheets/application/common.scss

```diff
@@ -266,3 +268,66 @@ header .container-fluid nav.navbar li.dropdown {
     line-height: 1.5;
   }
 }
+.body-content {
+  width: 92%;
+  margin-top: 5%;
+  margin-right: auto;
+  margin-left: auto;
+}
+.body-content-element {
+  padding-bottom: 4%;
+  border-radius: 5px;
+  margin-bottom: 3%;
+  box-shadow: 0 0 20px 0 rgba(0,0,0,0.2),0 5px 5px 0 rgba(0,0,0,0.24);
+}
+.content-title {
+  padding: 1% 0%;
+  margin-bottom: 0;
+  line-height: 1.3;
+  text-align: center;
+  font-size: 250%;
+  background-color: #ffaeda;
+  border-radius: 5px;
+  width: 50%;
+  color: #3E4659;
+}
+.content-subtitle {
+  text-align: center;
+  color: #01b2ac;
```

Comment:
we define colors in app/assets/stylesheets/variables.scss so we can be consistent with them

Reply:
Ok I'll follow this pattern.